docs/dex-specification.md

```diff
@@ -672,7 +674,7 @@ The function returns the address of a new staking pool contract.
 
 #### `StakingInitializable`
 
-The staking factory contract (`StakingFactory`) deploys the `StakingInitializable` contract when a new staking pool is deployed. This contract initializes a new pool.
+The staking factory contract (`StakingFactory`) deploys the `StakingInitializable` contract when a new staking pool is deployed. This contract initializes a new pool. Note that the reward tokens have to be sent manually after the deployment.
```

Comment:
StakingInitializable` contract is a staking pool, so **`deploys the `StakingInitializable` contract when a new staking pool is deployed`** is a bit wrong. The staking factory contract (`StakingFactory`) deploys the `StakingInitializable` contract which is a new staking pool. Staking factory initializes the pool after it is deployed in the deployPool function. Btw, deployPool params have to be reordered in the specification (the multisig address is the last parameter). If the reward tokens are not sent to the pool before startBlock the pool will not correctly operate `deposit` and `withdraw` functions. Each user will be able to deposit only for their first time (since they don't have available rewards yet) and since then these (`deposit` and `withdraw`) functions will fail with 'TransferHelper: TRANSFER_FAILED', hence to the pool tries to send available rewards to the user who calls them. So the reward tokens have to be sent to the pool in the time between after it gets deployed and startBlock (block when the pool starts).